addon/components/chart-component.js

```diff
@@ -28,19 +28,27 @@ export default Ember.Component.extend(ColorableMixin, ResizeHandlerMixin, {
   horizontalMarginRight: null,
 
   /**
+   * An array of the values in the data that is passed into the chart
+   * @type {Array<Number>}
+   */
+  allValues: Ember.computed('finishedData.@each.value', function() {
+    return this.get('finishedData').map((d) => d.value);
```

Comment:
I think I tried that and it didn't work because `finishedData` is a regular array